.github/workflows/pipeline.yml

```diff
@@ -240,3 +240,12 @@ jobs:
       - uses: actions/checkout@v3
       - name: "nix-tools build"
         run: cd ./nix-tools && .buildkite/nix-tools-build.sh
+
+  deadnix:
+    runs-on: [self-hosted, linux]
+    steps:
+      - uses: actions/checkout@v3
+      - run: nix run github:astro/deadnix -- --edit --no-lambda-pattern-names
+      - uses: yvan-sraka/action-suggester@master
```

Comment:
Is this really needed? Notice that you are running nix in the second step and then installing it again as part of the third step.

Reply:
The [PATCH I applied in my fork of the GHA](https://github.com/reviewdog/action-suggester/compare/master...yvan-sraka:action-suggester:master) is because, AFAIK, we're using custom NixOS job runners that would make the action fail otherwise https://github.com/reviewdog/action-suggester/issues/35 …

Reply:
The issue you link has now been closed. Is your patch still necessary?